embedchain/embedchain.py

```diff
@@ -67,11 +60,12 @@ def _get_loader(self, data_type):
         else:
             raise ValueError(f"Unsupported data type: {data_type}")
 
-    def _get_chunker(self, data_type):
+    def _get_chunker(self, data_type, url=None):
```

Comment:
no need for `url` here, let the chunker decide how to deal with url in `load_and_embed` method